app/build.gradle

```diff
@@ -1,10 +1,13 @@
+ext {
+    networkSurveyMessagingVersion = networkSurveyMessagingVersion
+}
 apply plugin: 'com.android.application'
 apply from: "../artifacts.gradle"
 
 android {
     compileSdkVersion 29
-    buildToolsVersion "29.0.3"
-    ndkVersion "21.3.6528147"
+    buildToolsVersion '29.0.2'
```

Comment:
I had to update the changes to fix the minify issue manually since I merged the develop branch and missed that difference. Also had to install ndk and it might've installed changed the build version in the process.

Reply:
Ahh that must be what happened, it automatically updated these two lines. Go ahead and revert this change. I seem to remember having problems with certain NDK versions where it would not compile the native lib correctly.